source/_data/glossary.yml

```diff
@@ -70,5 +72,7 @@
   description: "A [trigger](/docs/automation/trigger/) is a set of values or conditions of a platform that are defined to cause an automation to run."
 - topic: TTS
   description: "TTS ([text to speech](/integrations/tts)) allows Home Assistant to talk to you."
+- topic: Variables
+  description: "[Variables](/docs/scripts/#variables) are a named storage for a value that can be processed e.g. in a script."
```

Comment:
I think "storage" is a wrong them, as they are not stored at all.

Reply:
Changed. You are right. it is not classic persistent storage and this could be misunderstood.
See #23737